app/src/main/java/com/kuit/kuit6android/ui/home/component/CategoryRow.kt

```diff
@@ -0,0 +1,36 @@
+package com.kuit.kuit6android.ui.home.component
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.modifier.modifierLocalConsumer
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.kuit.kuit6android.R
+import com.kuit.kuit6android.ui.home.data.CategoryData
+
+
+@Composable
+fun CategoryRow(modifier: Modifier = Modifier) {
+    val categoryList = listOf(
+        CategoryData(imageId = R.drawable.img_donka,"돈까스"),
+        CategoryData(imageId = R.drawable.img_japanese,"일식"),
+        CategoryData(imageId = R.drawable.img_korean, "한식"),
+        CategoryData(imageId = R.drawable.img_chicken,"치킨"),
+        CategoryData(imageId = R.drawable.img_snack,"분식")
+    )
+    LazyRow (horizontalArrangement = Arrangement.spacedBy(18.dp)){
```

Comment:
```suggestion
    LazyRow(
        Modifier
            .fillMaxWidth()
            .padding(horizontal = 26.5.dp), horizontalArrangement = Arrangement.SpaceBetween
    ) {
```

이런식으로 spaceBy로만 여백을 주면 기기가 가로로 길어졌을 때, 가운데에 요소들이 너무 몰려있고, 좌우 여백은 너무 클 수 있기 때문에 SpaceBetween 사용하는 게 더 좋을 것 같습니다!